src/search-scenario/azext_search_scenario/constants.py

```diff
@@ -0,0 +1,48 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from enum import Enum
+
+
+class SearchScope(int, Enum):
+    All = 1
+    Scenario = 2
+    Command = 3
+
+    @staticmethod
+    def get_search_scope_by_name(name):
```

Comment:
Is it better to use the function name as `get(scope)`? because
- The name your class is `SearchScope` and it's redundant to add `search_scope` in function name.
- The value of the parameter is one of 3 scopes, so it's straightforward to use `scope` as the name of parameter.

Reply:
Done